style.css

```diff
@@ -13,9 +13,23 @@ body {
   background-color: #e5e5e5;
 }
 
+@media (max-width: 768px) {
+  .shapes-background {
+    position: absolute;
+    left: calc(50% - 375px / 2 + 2px);
+    width: 375px;
+    height: 748px;
+    background-image: url('./Images/header-shapes mobile@2x.svg');
+    background-position: 0 122px;
+    background-repeat: no-repeat;
+    background-color: #fff;
+    border-radius: 0 0 0 100px;
+  }
+}
+
```

Comment:
Wonderful job 👍👍 However, there are few things that still need to be fixed.
- [ ] As suggested by the previous reviewer, please make sure that your portfolio id fully responsive. Right now while switching from `mobile` to `desktop` version, your design is not responsive, however it is responsive after `768px`. See the video below for more details 🔽📷 This is because users have different screen sizes and it can be a bad user experience if a design is not responsive. And we don't want that Right? 🎯

https://user-images.githubusercontent.com/106119105/214983789-215cde6a-b0ba-4d7d-84b8-94ed6fb70058.mp4

For more clarity on responsive design, you can visit the following articles
- [HTML Responsive Web Design](https://www.w3schools.com/html/html_responsive.asp)
- [Responsive design](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Responsive_Design)